Meta/CMake/lagom_compile_options.cmake

```diff
@@ -1,9 +1,8 @@
 include(${CMAKE_CURRENT_LIST_DIR}/common_compile_options.cmake)
 
-add_cxx_compile_options(-Wno-maybe-uninitialized)
-add_cxx_compile_options(-Wno-shorten-64-to-32)
-
 if(NOT MSVC)
```

Comment:
disabling these warnings without enabling the equivalent warnings for the windows config isn't very useful.